sharness/t0050-service-migration.sh

```diff
@@ -0,0 +1,33 @@
+#! /bin/sh
+
+test_description="Test service migration v1 -> v2 and v2 -> v2"
+
+. lib/test-lib.sh
+
+test_ipfs_init
+cleanup test_clean_ipfs
+test_cluster_init
+test_create_v1State
+cleanup test_clean_cluster
+
+test_expect_success IPFS,CLUSTER "cluster-service state preserved by migration" '
+    cid=`docker exec ipfs sh -c "echo test | ipfs add -q"`
+    ipfs-cluster-ctl pin add "$cid" &> test4 && sleep 2 &&
+    kill -1 CLUSTER_D_PID && sleep 2 &&
+    backup-file=$( ls test-config/backups/ | head -n 1)
+    ipfs-cluster-service migration "test-config/backups/"$(backup-file) "test-config/ipfs-cluster-data"
```

Comment:
Are we forgetting to check if this command is successful ? It would seem that, in case of failure, it will go unnoticed

Reply:
if everything is not chained with `&&` and there are no returns, sharness will happily keep going forward.